lib/cdo/languages.rb

```diff
@@ -0,0 +1,16 @@
+require 'cdo/db'
+DB = PEGASUS_DB
+
+class Languages
+
+  @@table = DB[:cdo_languages]
+
+  def self.get_crowdin_languages()
+    @@table.select(:crowdin_code_s, :crowdin_name_s).where("crowdin_code_s != 'en'")
+  end
+
+end
+
+def fetch_crowdin_languages()
```

Comment:
Not sure why this is needed - you can just call Languages.get_crowdin_languages in target_languages.haml, right?